lms/djangoapps/teams/search_indexes.py

```diff
@@ -0,0 +1,95 @@
+""" Search index used to load data into elasticsearch"""
+
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from search.search_engine_base import SearchEngine
+
+from .serializers import CourseTeamSerializer, CourseTeam
+
+
+class CourseTeamIndexer(object):
+    """
+    Index object for CourseTeam model instances
+    """
+    INDEX_NAME = "course_team_index"
+    DOCUMENT_TYPE_NAME = "course_team"
+    ENABLE_SEARCH_KEY = "ENABLE_TEAMS_SEARCH"
+
+    def __init__(self, course_team):
+        self.course_team = course_team
+
+    def data(self):
+        """
+        Uses the CourseTeamSerializer to create a serialized course_team object
```

Comment:
Nit: can you use proper punctuation in docstrings? Same goes for the rest of this file.